doc/languages-frameworks/python.md

```diff
@@ -481,7 +481,7 @@ and the aliases
 #### `buildPythonPackage` function
 
 The `buildPythonPackage` function is implemented in
-`pkgs/development/python-modules/generic/default.nix`
+`pkgs/development/interpreters/python/buildpythonpackage.nix`
```

Comment:
actually, there should be dashes in between the words
http://hydra.nixos.org/build/37411318/download/1/nixpkgs/manual.html#sec-organisation